tests/cli_commands/test_prepare_env.py

```diff
@@ -19,7 +19,7 @@ class GenHomeProfilesCommandTestCase(unittest.TestCase):
             "outputs": {
                 "env_execution": {
                     "method": "service-account",
-                    "project": "example-project",
+                    "project": "exampleproject",
```

Comment:
Was there reason than a typo behind this change?

Reply:
You cannot provide project names with `-` anymore - dbt wizard simply fails (tried different options).